server/bin/pbench-server-prep-shim-002.py

```diff
@@ -261,39 +124,23 @@ def process_tb(config, logger, receive_dir, qdir_md5, duplicates, errors):
             )
 
         try:
-            os.symlink((dest / resultname), (dest / "TODO" / resultname))
+            os.remove(tb)
         except Exception as exc:
-            logger.error("{}: Error in creation of symlink. '{}'", config.TS, exc)
-            # if we fail to make the link, we quarantine the (already moved)
-            # tarball and .md5.
-            quarantine(
-                (errors / controller), logger, (dest / tb), (dest / tbmd5),
+            logger.error(
+                "{}: Warning: cleanup of successful copy operation failed: '{}'",
+                config.TS,
+                exc,
             )
-            nerrs += 1
-            continue
 
         ntbs += 1
 
-        try:
-            if dataset:
-                dataset.advance(States.UPLOADED)
-        except Exception:
-            logger.exception("Unable to finalize {}", dataset)
-
         nstatus = f"{nstatus}{config.TS}: processed {tb}\n"
         logger.info(f"{tb.name}: OK")
 
-    return Results(
-        nstatus=nstatus,
-        ntotal=ntotal,
-        ntbs=ntbs,
-        nquarantined=nquarantined,
-        ndups=ndups,
-        nerrs=nerrs,
-    )
+    return Results(nerrs=nerrs, nstatus=nstatus, ntotal=ntotal, ntbs=ntbs,)
```

Comment:
It seems slightly odd to report the number of copied tarballs, the number of errors, _and_ the total number of attempts -- I believe that from any two of those we can calculate the third, which makes measuring it directly and storing it kind of a waste.

Reply:
(Also, we don't need (or want) the trailing comma in the argument list.)

Reply:
I removed the nerrs. and will be calculating that with the rest of the collected details.